Web/starter_project_barbies/components/team/TeamMemberCard.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactNode } from "react";
+import {
+  BsFacebook,
+  BsLinkedin,
+  BsInstagram,
+  BsEmojiSmileUpsideDown,
+} from 'react-icons/bs'
+import Image from 'next/image'
+import { TeamMember } from '@/types/team'
+
+export type TeamMemberCardProps = {
+  teamMember: TeamMember
+}
+
+const socialMediaIcon: { [index: string]: ReactNode  } = {
+  facebook: <BsFacebook className="fill-gray-400 w-6 h-6" />,
+  linkedin: <BsLinkedin className="fill-gray-400 w-6 h-6" />,
+  instagram: <BsInstagram className="fill-gray-400 w-6 h-6" />,
+  default: <BsEmojiSmileUpsideDown className="fill-gray-400 w-6 h-6" />,
+}
+
+const TeamMemberCard: React.FC<TeamMemberCardProps> = ({
+  teamMember: { name, jobTitle, description, profileImg, socialMediaLinks },
+}: TeamMemberCardProps) => {
+  return (
+    <div className="box-content font-sans w-72 mt-4 text-center shadow-md shadow-gray-200 rounded-md py-2 m-auto">
+      <div className="container overflow-hidden my-4">
+        <Image
+          className="rounded-full mx-auto w-32"
+          width={200}
+          height={200}
+          src={profileImg}
+          alt={name}
+        />
+      </div>
+
+      <div className="m-4">
+        <h3 className="text-2xl m-3 font-extrabold tracking-widest uppercase">
+          {name}
+        </h3>
+        <p className="m-3 tracking-widest text-base uppercase">{jobTitle}</p>
+        <p className="text-sm mt-5 mb-3 text-gray-400 font-light px-4 tracking-wide">
```

Comment:
[nit] its best if you use the colors you've already defined in your tailwind.config file to make it more uniform.